demo/index.html

```diff
@@ -21,32 +21,57 @@
 
   <link rel="import" href="../iron-input.html">
 
-  <link rel="stylesheet" href="../../paper-styles/demo.css">
+  <link href="../../paper-styles/demo-pages.html" rel="import">
+    <link href="../../paper-button/paper-button.html" rel="import">
 
-</head>
-<body>
-
-  <template is="dom-bind">
+  <link href="../../paper-styles/paper-styles.html" rel="import">
 
-    <section>
+  <style is="custom-style">
+    section {
+      @apply(--paper-font-body1);
 
-      <p>
-        <input is="iron-input" bind-value="{{bindValue}}" value="{{value::input}}">
-        <br>
-        bind to <code>bind-value</code>: <span>[[bindValue]]</span>
-        <br>
-        bind to <code>value::input</code>: <span>{{value}}</span>
-      </p>
+      line-height: 40px;
+    }
 
-      <p on-click="setValue">
-        set bind-value to: <input> <button value="bindValue">set</button>
-        <br>
-        set value to: <input> <button value="value">set</button>
-      </p>
+    code {
+      color: var(--google-grey-700);
+    }
 
-    </section>
+    input[is=iron-input] {
+      width: 100%;
+      box-sizing: border-box;
+    }
 
-  </template>
+    span {
+      font-weight: bold;
+    }
+    
+    input, button {
+      font-size: 20px;
+      padding: 0.2em;
+    }
+  </style>
+</head>
+<body>
+  <div class="vertical-section vertical-section-container centered">
+    <template is="dom-bind">
+      <section>
```

Comment:
do you need the `<section>`? I think the `<vertical-section>` is probably enough